Changelog.md

```diff
@@ -111,6 +111,7 @@ Bugfixes:
  * Type Checker: Report error when using structs in events without experimental ABIEncoderV2. This used to crash or log the wrong values.
  * Type Checker: Report error when using indexed structs in events with experimental ABIEncoderV2. This used to log wrong values.
  * Type Checker: Dynamic types as key for public mappings return error instead of assertion fail.
+ * Type Checker: Fix crash when array index value is too large.
```

Comment:
Perhaps better to use the word "internal error" - "crash" always sounds like a segmentation violation to me.